eth/gaspricemonitor.go

```diff
@@ -112,6 +112,9 @@ func (gpm *GasPriceMonitor) Stop() error {
 	gpm.cancel = nil
 	gpm.polling = false
 
+	// Close the update channel when gpm is stopped
+	close(gpm.update)
```

Comment:
Can we add a test that checks for this close?

Reply:
added in https://github.com/livepeer/go-livepeer/pull/991/commits/b9d2fae0b21dd245bd209c43f55a989f8dd36f34